md/bonita-as-windows-service.md

```diff
@@ -0,0 +1,100 @@
+# How to install Bonita as a service on Windows
+
+If you are a Windows server administrator you know how useful can be to have softwares installed as a windows service.
+In this tutorial we show how to install Tomcat with Bonita configured as a windows service.
+
+## Software needed
+
+For this tutorial you will need the following softwares:
+* Tomcat
+* Bonita deploy Bundle
+* Postgresql Database (or another database supported by Bonita)
+Please verify the version supported from the Support page
+
+## Download and installation
+
+* **_Download_** apache-tomcat-x.y.exe from Tomcat website: https://tomcat.apache.org/
+* **_Install_** apache-tomcat-x.y.exe. From now on we'll call the installation path as %TOMCAT_INSTALL_FOLDER%
+* **_Verify_** that Tomcat is well installed browsing the following URL: http://localhost:8080
+
+![Tomcat home page](images/bonita-as-windows-service/tomcatHome.png)
```

Comment:
⚠️ the image is very large, can it be reduced?
in addition, the version shown here in not the one supported for Bonita 7.4

Reply:
how to do to have different images for the different product versions? Maybe we can just update the image during resizing for removing the version, leave only:
![image](https://user-images.githubusercontent.com/7021326/46962201-d4e72e80-d0a2-11e8-94c6-c774dd7afa4d.png)

Reply:
the image has to be resized, I agree :-)

Reply:
I have removed the version in the image